assets/src/components/minimized-display.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import dayjs from "dayjs";
+import * as PropTypes from "prop-types";
+import "./minimized-display.scss";
+
+/**
+ * Minimized display component.
+ *
+ * @param {object} props Props.
+ * @param {object} props.validUrlParams Validated parameters.
+ * @param {Function} props.setDisplayState Set display state function.
+ * @param {object} props.urlResource Resource fetched from URL.
+ * @returns {string} Calendar header component.
+ */
+function MinimizedDisplay({ validUrlParams, setDisplayState, urlResource }) {
+  const onChangeBooking = () => {
+    setDisplayState("maximized");
+  };
+
+  return (
+    <div className="col-md-12">
+      <div className="row">
+        {urlResource && (
+          <div className="minimized-display col-md-12">
+            <div>
+              <span className="location">{urlResource.location}</span>
+              <span className="subject">{urlResource.resourceName}</span>
+            </div>
+            <div>
+              <span>
+                {dayjs(validUrlParams.get("from")).format("DD/MM/YYYY - HH:mm")}
```

Comment:
I would probably add a function here to handle formatting, so it can be used with "from" and "to".

```
const formatUrlParam = (dateString) => dayjs(dateString).format("DD/MM/YYYY - HH:mm");
```

and call it with
```
{formatDate(validUrlParams.get("from"))}
```